src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorController.ts

```diff
@@ -134,6 +136,17 @@ export class InteractiveEditorController implements IEditorContribution {
 		return INTERACTIVE_EDITOR_ID;
 	}
 
+	private _getMode(): EditMode {
+		let editMode: EditMode = this._configurationService.getValue('interactiveEditor.editMode');
+		const isDefault = editMode === EditMode.LivePreview;
```

Comment:
Better use `IConfigurationService.inspect` and look at `IConfigurationValue#defaultValue` and `value`. That avoid hardcoding the (current) default and it will account for the fact where users explicitly configure live preview to be their choice.

Reply:
Sample: https://github.com/microsoft/vscode/blob/efb49cc271d6eea5634ac395e36e1e4cd108a447/src/vs/workbench/browser/parts/titlebar/windowTitle.ts#L263-L267

Reply:
I will create a follow up PR